check-servers.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh -e
 
-[ "$ENABLE_IPv4" = "1" ] && wget -4qSO /dev/null http://127.0.0.1:8080/adv
-[ "$ENABLE_IPv6" = "1" ] && wget -6qSO /dev/null http://[::1]:8080/adv
+[ "$ENABLE_IPv4" = "1" ] && $(wget -4qSO /dev/null http://127.0.0.1:8080/adv || exit 1)
```

Comment:
I think it might be cleaner to use an `if-then-else` block instead of using subshell expansions and `exit`s.

```suggestion
if [ "$ENABLE_IPv4" = "1" ]; then
  wget -4qSO /dev/null http://127.0.0.1:8080/adv
fi
```

(same comment on the `ENABLE_IPv6` case below.